src/components/metrics/card/index.js

```diff
@@ -182,7 +184,20 @@ export default class MetricCard extends Component {
     this.props.changeMetric({id: this._id(), name})
   }
 
-  onChangeGuesstimateDescription(description) {
+  onChangeGuesstimateDescription(rawDescription) {
+    const fullRegex = new RegExp(`\\[.+\\]\\(${URL_REGEX.source}\\)`, 'g')
+    let description = ''
+    let prevIndex = 0
+    let matchArr
+    while ((matchArr = fullRegex.exec(rawDescription)) !== null) {
+      const partial = rawDescription.slice(prevIndex, matchArr.index)
+      description += partial.replace(URL_REGEX, match => `[${match}](${match})`)
+      description += matchArr[0]
+      prevIndex = matchArr.index + matchArr[0].length
+    }
+    const partial = rawDescription.slice(prevIndex)
+    description += partial.replace(URL_REGEX, match => `[${match}](${match})`)
```

Comment:
can the url changing part of this be moved into a separate function?  I imagine testing could be useful for that, but it's not a huge concern now.